src/python/pants/backend/python/goals/coverage_py_integration_test.py

```diff
@@ -69,6 +69,8 @@ def test_add():
         "src/python/core/BUILD": "python_sources()",
```

Comment:
That's what I tried at the start, but when I was debugging the content I got in the `get_namespace_value_from_config` function I added, it was the content from the project's `pyproject.toml`, never from the file I created. Any ideas how to solve this would be appreciated.

Reply:
Created a separate parametrized test for the different cases, not to bloat something already existing.